products/mxlinux.md

```diff
@@ -0,0 +1,83 @@
+---
+permalink: /mxlinux
+alternate_urls:
+-   /mx_linux
+title: MX Linux
+category: os
+releasePolicyLink: https://mxlinux.org/release-cycle/
+iconSlug: mxlinux
+activeSupportColumn: true
+versionCommand: cat /etc/os-release
+releaseColumn: false
+releaseDateColumn: true
+realseLabel: "__RELEASE_CYCLE__ (__CODENAME__)"
+releasePolicyLink: https://mxlinux.org/release-cycle/
+purls:
+-   purl: pkg:os/mxlinux
+auto:
+-   distrowatch: devuan
+    regex: '^Distribution Release: MX Linux (?P<major>\d)\.(?P<minor>\d)$'
```

Comment:
```suggestion
    regex: '^Distribution Release: MX Linux (?P<major>\d+)(?:\.(?P<minor>\d+))?(?:\.(?P<patch>\d+))?'
```

- The minor part is optional
- Both major/minor/patch can be multiple digits
- There can be patch versions (not listed on DW yet)

I used this to validate the regex: https://regex101.com/r/VjVvlu/1

Got the list of releases from the [DW page](https://distrowatch.com/index.php?distribution=mx) with the following in the web console:

`copy(Array.from(document.querySelectorAll('td.NewsHeadline')).map((x)=>{return x.innerText}))`

It still misses the KDE 19.2 release, but that's okay since we can only track 19.2 once anyway.